webSocket/tests/integrationTests/clientAndServer_test.go

```diff
@@ -0,0 +1,189 @@
+package integrationTests
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core/mock"
+	"github.com/multiversx/mx-chain-core-go/testscommon"
+	"github.com/multiversx/mx-chain-core-go/webSocket/data"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStartServerAddClientAndSendData(t *testing.T) {
+	url := "localhost:8833"
```

Comment:
this might seldom fail if the port is already taken, for example, it might sometimes fail on github actions server
I think we already have a function that searches for an empty port
https://github.com/multiversx/mx-chain-p2p-go/blob/b6fe55be463cdd67bf119d9e1f7eacc2da3053ad/libp2p/ports.go#L74

Valid on all occurrences on this file.

Reply:
will to it on the next PR